src/frontend/src/components/common/Modal/_modal.scss

```diff
@@ -5,17 +5,47 @@
   width: 100%;
   height: 100%;
   background: rgba(0, 0, 0, 0.6);
+  border-radius: 5px;
 
+  .modal-title {
+    display: block;
+    font-weight: bold;
+    position: relative;
+    text-align: center;
+    padding-top: 1em;
+    color: black;
+  }
+
+  .horizontal-divider {
```

Comment:
There is a class already using this name so I would suggest a different name if the current implemented appearance doesn't meet your needs.

Reply:
create a new selectot for modal